### PR TITLE
Fix MSVC warning C4373 (virtual function overrides).

### PR DESCRIPTION
**Pull request by ptoharia**

### CODE REVIEW

**Comment by ptoharia** [comment]
Servus' pull request needed to remove some MSVC errors. Still some other errors.

File: brion/plugin/compartmentReportBinary.cpp [hunk 1]
```diff
@@ -204,7 +204,7 @@ size_t CompartmentReportBinary::getFrameSize() const
     return _subtarget ? _comps : _header.numCompartments;
 }
 
-floatsPtr CompartmentReportBinary::loadFrame( const float timestamp ) const
```
Comment (eile) [inline brion/plugin/compartmentReportBinary.cpp]: Should stay const here. Did you get a warning? If yes, which one?

Comment (ptoharia) [inline brion/plugin/compartmentReportBinary.cpp]: I was getting the virtual override and I changed all the prototypes, both in .h and .cpp files. I guess I should have changed only the headers, right? I'll roll back cpp changes if so

Comment (eile) [inline brion/plugin/compartmentReportBinary.cpp]: Yep. I don't have the link at hand, but basically the header is for users (const on by-value is pointless) and the impl is for developers (const means the copy param is not modified)

**Comment by ptoharia** [comment]
@eile rolled back cpps

**Comment by eile** [comment]
Retest this please.

**Comment by eile** [comment]
+1

**Pull request merged**